docs/index.html

```diff
@@ -0,0 +1,117 @@
+        </div>
+
+        <h2 class="walkthrough-title">
+            Mark Item Completed
+        </h2>
+        <h5 class="step-title">
+            Task list with uncompleted task
+        </h5>
+        <div class="step">
+            <iframe src="uncompletedTask.html" style="border:1px solid black;" width="320" height="640"></iframe>
+            <p>This shows the task list with an uncompleted task. Checking the circle will complete the task.</p>
+        </div>
+        <h5 class="step-title">
+            Task list with completed task
+        </h5>
+        <div class="step">
+            <iframe src="main.html" style="border:1px solid black;" width="320" height="640"></iframe>
+            <p>After checking the uncompleted task, this is what the task list looks like.</p>
+        </div>
+
+        <h2 class="walkthrough-title">
+            Renaming Item
+        </h2>
+        <h5 class="step-title">
+            Task list with item with old undesirable name
+        </h5>
+        <div class="step">
+            <iframe src="badNameTask.html" style="border:1px solid black;" width="320" height="640"></iframe>
+            <p>This shows the task list with a item named "Text John" that should be renamed. The task's text can be clicked to edit it.</p>
+        </div>
+        <h5 class="step-title">
+            Task list with newly revised name
+        </h5>
+        <div class="step">
+            <iframe src="main.html" style="border:1px solid black;" width="320" height="640"></iframe>
+            <p>After renaming the old item "Text John", this is what the task list looks like.</p>
+        </div>
+
+        <h2 class="walkthrough-title">
+            Show Only Uncompleted Tasks
+        </h2>
+        <h5 class="step-title">
+            Task list with completed task
+        </h5>
+        <div class="step">
+            <iframe src="main.html" style="border:1px solid black;" width="320" height="640"></iframe>
+            <p>This shows the task list with one completed task. The eye button in the upper-left corner performs the action of showing only uncompleted tasks.</p>
+        </div>
+        <h5 class="step-title">
+            Task list after hiding completed tasks
+        </h5>
+        <div class="step">
+            <iframe src="showUncompleted.html" style="border:1px solid black;" width="320" height="640"></iframe>
+            <p>After hiding all of the completed tasks, this is what the task list looks like.</p>
+        </div>
+
+
+        <h2 class="walkthrough-title">
+            Delete Completed Tasks
+        </h2>
+        <h5 class="step-title">
+            Task list with completed item
+        </h5>
+        <div class="step">
+            <iframe src="main.html" style="border:1px solid black;" width="320" height="640"></iframe>
+            <p>This shows the task list with one completed task. The trash button in the upper-right corner performs the action of deleting completed tasks.</p>
```

Comment:
Wouldn't it be easy to accidentally delete? Would some form of confirmation be in order?